server/protocols/jmap-rfc-8621/doc/README.md

```diff
@@ -0,0 +1,11 @@
+Annotated JMAP documentation
+============================
+
+This directory contains annotated JMAP documentation as found on https://jmap.io/.
+
+Officially finalized specifications so far regarding JMAP are:
+
+* [The core protocol](https://jmap.io/spec-core.html) [[RFC 8620](https://tools.ietf.org/html/rfc8620)]
+* [JMAP Mail](https://jmap.io/spec-mail.html) [[RFC 8621](https://tools.ietf.org/html/rfc8621)]
+
+Annotations aim at tracking implementation progress in James project.
```

Comment:
Can we have a note on how to use the Makefile ?

Reply:
Not sure if we should keep the Makefile as well TBH... I remember it was a pain and I had to modify it a bit to make it work for geenrating the quota specs

Reply:
Then let's drop it!

Reply:
What's the plan for rendering: github?

Reply:
Not sure... do you need a Docker image like for draft?

Reply:
If needed I have an updated Makefile, but I agree it's a pain and we can drop it. The interest of this Makefile is to transform Markdown files into RFC (xml/txt/html) files. It has no interest for us. If the github rendering is ok to show us the annotations indicating the status of the development, it would already be great.

Reply:
Not very readable unfortunately...

![Capture d’écran de 2020-07-28 14-43-53](https://user-images.githubusercontent.com/1398581/88666946-08ad1000-d0e1-11ea-9bd3-cbb2641d5108.png)
![Capture d’écran de 2020-07-28 14-44-30](https://user-images.githubusercontent.com/1398581/88666948-0945a680-d0e1-11ea-881d-c8f6f3d53a85.png)

Reply:
I will try to check if we can't do like the draft specs, with a Docker image, didn't seem so bad